platform/darwin/test/MGLNSStringAdditionsTests.m

```diff
@@ -19,7 +19,7 @@ - (void)testTitleCasedString {
 
     XCTAssertEqualObjects([@"Improve the map" mgl_titleCasedStringWithLocale:locale], @"Improve the Map");
     XCTAssertEqualObjects([@"Improve The Map" mgl_titleCasedStringWithLocale:locale], @"Improve The Map");
-
+    
```

Comment:
Nit: whitespace only change

Reply:
This was in the original commit, so I’ve just let it be. 🤷‍♂️